lib/jazzy/sourcekitten.rb

```diff
@@ -282,7 +283,11 @@ def self.process_undocumented_token(doc, declaration)
       if objc || should_mark_undocumented(doc['key.kind'], filepath)
         @stats.add_undocumented(declaration)
         return nil if @skip_undocumented
-        declaration.abstract = @undocumented_abstract
```

Comment:
...and this can be `= undocumented_abstract` without ifs to upset rubocop.

Reply:
thank you :)